frontend/src/components/headerSection/header.css

```diff
@@ -4,15 +4,16 @@
 }
 
 .app-header {
-    height: calc(15vh + 64px);
+    min-height: 70vh;
```

Comment:
Did you test it on mobiles and portrait monitors?

Reply:
Good point with the portrait monitor... It's not great. Let me fix that.

Reply:
Only 4k portrait view has real problems so I'll file an issue for this and fix later
![image](https://user-images.githubusercontent.com/6146988/130184039-871a7403-8ff8-41f3-824c-7a386469cfc1.png)